### ISSUE TITLE
Metapad should probably be identified as "Metapad" in the task manager

### ISSUE THREAD

**Issue by AlisterH** [sobitcorp/metapad#4]
Is this "Fast, tiny..." string is being used somewhere where it should just say "Metapad"?
The line underneath is also unusual - other apps that show the name of the open file don't append the app name afterwards.
![image](https://user-images.githubusercontent.com/714600/161709137-56cad5c5-54c9-4797-bdaf-53e4d4d11b7f.png)

**Comment by sobitcorp** [comment]
I noticed this as well, and agree. This is because I added a VERSIONINFO resource to the executable. The FileDescription string needs to be moved to Comment. I will fix this on next release.
The line underneath is just the window caption. Metapad always added its name after the filename. Doesn't notepad do this as well?

**Comment by AlisterH** [comment]
> The line underneath is just the window caption. Metapad always added its name after the filename. Doesn't notepad do this as well?

Yes.  I thought I saw some other programs that don't, but maybe I'm wrong and it is only explorer that doesn't.
